examples/apps/django2/django2sample/settings.py

```diff
@@ -0,0 +1,122 @@
+"""
```

Comment:
I generally prefer to name examples by what they actually do, rather than like "django2sample", because over time there might be any number of django2 samples.

Reply:
I agree with you, except here you might be missing that django2sample is a django project and contains not much of interest, except some necessary stuff to make the apps available...it's the apps that are the examples, and which should have meaningful names. (There can be many apps. Right now there's just a "sliders" app.)

You might not have been confused if there were less boilerplate, which relates to this comment in my PR description:

> Perhaps in the future, we could provide just the apps, and on CI/in the installation instructions, have a diff that gets applied to the result of running django-admin startproject in order to make the apps available in a project.

Meanwhile, maybe it would be clearer as django2samples? Or django2examples?

(I called it django2 because we may need versions of the apps for at least one previous django version. I'm not sure about that yet.)

Reply:
Maybe you can see how to simplify the paths a bit if I clarify more?

```
examples/apps/django2/django2sample/
examples/apps/django2/sliders/
examples/apps/django2/app1/
examples/apps/django2/app2/
...
``` 

`examples/apps` because there'll be other types of example (e.g. notebooks...).
`examples/apps/django2` because there could be flask, django1, etc

The django apps in examples/apps/django2 (sliders, app1, app2, ...) are standalone and could be included in any django project. django2sample is just *a* (minimal) django project, so that there's something easy for someone to run without having to do anything other than conda install django=2.

Reply:
I agree it would be nice to build the project from a few key diffs rather than capturing all the boilerplate, but either way, maybe it could be "django2project" or just "project" (as it's in the django2 directory already).  Not sure how to distinguish app1 and app2, but whatever the difference between those two is, they should be named accordingly.

Reply:
Sorry, app1 and app2 were just placeholders to demonstrate the point of multiple apps, with one project to allow any/all of the apps to be run - i.e. app1 and app2 don't yet exist. 

I think "project" would be ok for the project name, and I'll use that. I suggested demoproject above, but just project seems clearer given the overall path.